### PR TITLE
Add server configuration files

### PR DESCRIPTION
**Pull request by sandcha**
Connected to openfisca/openfisca-core/issues/568

Adds configuration files for:
* www.openfisca.fr
* api-test.openfisca.fr
  - Set a 302 from api-test to fr.openfisca.org/api/v2
* fr.openfisca.org
  - Link fr.openfisca.org to www.openfisca.fr content
  - Add fr.openfisca.org/api/v2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,12 @@ cd /home/openfisca
 git clone git@github.com:openfisca/openfisca-ops.git
 ```
 
-Each sub-directory has its own README.md file.
+For more information on `auto-update` sub-directories, please see their own README.md files.
+
+## Server configuration
+
+Main nginx services' configuration files are stored in a directory named after a server name.
```
Comment (Anna-Livia) [inline README.md]: The main nginx services' configuration files are stored in a directory named after a server name.

File: README.md [hunk 2]
```diff
@@ -13,7 +13,12 @@ cd /home/openfisca
 git clone git@github.com:openfisca/openfisca-ops.git
 ```
 
-Each sub-directory has its own README.md file.
+For more information on `auto-update` sub-directories, please see their own README.md files.
+
+## Server configuration
+
+Main nginx services' configuration files are stored in a directory named after a server name.  
+For example: `www.openfisca.fr` configuration file is `openfisca-ops/www.openfisca.fr/www.openfisca.fr.conf`
```
Comment (Anna-Livia) [inline README.md]: For example: `www.openfisca.fr` 's configuration file is in `openfisca-ops/www.openfisca.fr/www.openfisca.fr.conf`

File: fr.openfisca.org/fr.openfisca.org.conf [hunk 3]
```diff
@@ -0,0 +1,26 @@
+server {
+	listen 80;
+	server_name fr.openfisca.org;
+
+   	access_log /var/log/nginx/fr.openfisca.org-access.log;
+        error_log /var/log/nginx/fr.openfisca.org-error.log;
```
Comment (Anna-Livia) [inline fr.openfisca.org/fr.openfisca.org.conf]: White space at the start

File: fr.openfisca.org/fr.openfisca.org.conf [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+        location /.well-known {
+            root /tmp/renew-webroot;
+        }
+
+	location /api/v2 {
+		proxy_pass http://localhost:6000/;
+                proxy_set_header Host $http_host;
```
Comment (Anna-Livia) [inline fr.openfisca.org/fr.openfisca.org.conf]: white space (lines bellow also)

File: www.openfisca.fr/www.openfisca.fr.conf [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+	error_log /var/log/nginx/www.openfisca.fr-error.log;
+
+	location / {
+		proxy_pass http://localhost:2010;
+		proxy_set_header Host $http_host;
+		proxy_http_version 1.1;
+		proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+	}
+
+	location /robots.txt {
```
Comment (Anna-Livia) [inline www.openfisca.fr/www.openfisca.fr.conf]: Should we change the robot config so it does not reference openfisca.fr but openfisca.org instead ?

File: www.openfisca.fr/www.openfisca.fr.conf [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+	listen 443 ssl;
+	server_name www.openfisca.fr;
+
+        ssl_certificate /etc/letsencrypt/live/www.openfisca.fr-0001/fullchain.pem;
+        ssl_certificate_key /etc/letsencrypt/live/www.openfisca.fr-0001/privkey.pem;
+
+	access_log /var/log/nginx/www.openfisca.fr-access.log;
+	error_log /var/log/nginx/www.openfisca.fr-error.log;
+
+	location / {
```
Comment (Anna-Livia) [inline www.openfisca.fr/www.openfisca.fr.conf]: Why isn't there à 301 redirect here ?

File: README.md (hunk 2 above)
Comment (sandcha) [reply README.md]: 's added. 
> www.openfisca.fr.conf is the configuration file itself.

File: www.openfisca.fr/www.openfisca.fr.conf (hunk 6 above)
Comment (sandcha) [reply www.openfisca.fr/www.openfisca.fr.conf]: It redirects from http to httpS.

File: README.md [hunk 7]
```diff
@@ -13,7 +13,12 @@ cd /home/openfisca
 git clone git@github.com:openfisca/openfisca-ops.git
 ```
 
-Each sub-directory has its own README.md file.
+For more information on `auto-update` sub-directories, please see their own README.md files.
+
+## Server configuration
+
+The main nginx services' configuration files are stored in a directory named after a server name.
```
Comment (fpagnoux) [inline README.md]: - `nginx services` does not mean much : services are processes that are maintained alive by the OS. Nginx is just a reverse proxy that dispatch requests to the right application.
- `a server name` is ambiguous. Are we talking about a physical server ? You probably meant a domain name.

File: api-test.openfisca.fr/api-test.openfisca.fr.conf [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+	server_name api-test.openfisca.fr;
+
+	ssl_certificate /etc/letsencrypt/live/api-test.openfisca.fr/fullchain.pem;
+	ssl_certificate_key /etc/letsencrypt/live/api-test.openfisca.fr/privkey.pem;
+
+	access_log /var/log/nginx/api-test.openfisca.fr-access.log;
+	error_log /var/log/nginx/api-test.openfisca.fr-error.log;
+
+	location / {
+		proxy_pass http://localhost:6000;
```
Comment (fpagnoux) [inline api-test.openfisca.fr/api-test.openfisca.fr.conf]: As written on the issue : 

> Use a 302 to redirect api-test vers fr.openfisca.org/api

The traffic coming to `https://api-test.openfisca.fr` should also be redirected to `http://fr.openfisca.org/api/v2`.

File: www.openfisca.fr/www.openfisca.fr.conf [hunk 9]
```diff
@@ -0,0 +1,63 @@
+server {
```
Comment (fpagnoux) [inline www.openfisca.fr/www.openfisca.fr.conf]: Has we changed anything to this file 🤔  ?

File: www.openfisca.fr/www.openfisca.fr.conf (hunk 6 above)
Comment (fpagnoux) [reply www.openfisca.fr/www.openfisca.fr.conf]: All the other block mean to redirect the requests coming to `http://openfisca.fr`, `http://www.openfisca.fr` and `https://openfisca.fr` towards `https://www.openfisca.fr`.

This block indicates where `https://www.openfisca.fr` should actually find the application (`http://localhost:2010`).

File: www.openfisca.fr/www.openfisca.fr.conf (hunk 5 above)
Comment (fpagnoux) [reply www.openfisca.fr/www.openfisca.fr.conf]: That can be discussed in #13, but we're not there yet ;).

File: www.openfisca.fr/www.openfisca.fr.conf (hunk 9 above)
Comment (sandcha) [reply www.openfisca.fr/www.openfisca.fr.conf]: Nothing changed.

**Pull request merged**